packages/side-nav/src/SideNav.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { ulStyleOverrides } from './styles';
+
+const sideNavStyles = css`
+  width: 200px;
+`;
+
+interface SideNavProps {
+  className?: string;
+  children: React.ReactNode;
```

Comment:
The props are currently pretty simple, but probably worth adding TSDoc annotations here for a consistent DX.

Reply:
Yup, part of my current work is consistency amongst all forms of docs